python/glow/gwas/logistic_regression.py

```diff
@@ -0,0 +1,185 @@
+from pyspark.sql.types import StringType, StructField
+from typing import Any, Optional
+import pandas as pd
+import numpy as np
+from pyspark.sql import DataFrame
+import statsmodels.api as sm
+from dataclasses import dataclass
+from typeguard import typechecked
+from nptyping import Float, NDArray
+from scipy import stats
+import opt_einsum as oe
+from . import functions as gwas_fx
+from .functions import _VALUES_COLUMN_NAME
+
+__all__ = ['logistic_regression']
+
+fallback_none = 'none'
+
+
+@typechecked
+def logistic_regression(
+        genotype_df: DataFrame,
+        phenotype_df: pd.DataFrame,
+        covariate_df: pd.DataFrame = pd.DataFrame({}),
+        offset_df: pd.DataFrame = pd.DataFrame({}),
+        # TODO: fallback is probably not the best name
```

Comment:
In addition to `fallback` (I propose `correction` as an alternative name), we should expose a parallel to `pvalue_threshold` below which we perform the correction.